transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -429,4 +424,12 @@ private boolean getPinEventExecutorPerGroup() {
         return pinEventExecutor;
     }
 
+    private ChannelConfig setWakeupOnWrite(boolean wakeupOnWrite) {
+        this.wakeupOnWrite = wakeupOnWrite;
+        return this;
+    }
+
+    private boolean getWakeupOnWrite() {
```

Comment:
This is a private method and there exists another method `getPinEventExecutorPerGroup()` which returns a boolean. I can change both but not sure if it is needed.